sharing/src/main/scala/io/delta/sharing/spark/DeltaSharingUtils.scala

```diff
@@ -58,6 +61,61 @@ object DeltaSharingUtils extends Logging {
       metadata: model.DeltaSharingMetadata
   )
 
+  def getDeltaSharingTableMetadata(
```

Comment:
can you add some docs here and also the below two APIs do they need to be public?

Reply:
removed this one, which is in draft, to be used for streaming response.

Reply:
added comment for the one below.